scripts/snap

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "FAKE SNAP: Trying to execute 'snap $*' but snap are not supported on Docker, skipping it sorry."
+
+# Trying to guess the binary name, just to make the script exit succesfully.
+touch /usr/local/bin/"${2}" && chmod +x /usr/local/bin/"${2}"
```

Comment:
rather than installing a fake binary, how about just showing the message and returning? This way we won't have fake binaries hanging around (potentially conflicting with real ones)

Reply:
Hi David! We need the fake binary hanging around because the installers then check if the command exists.

Reply:
Ok. How about deleting them at the end in the dockerfile until the upstream PR is accepted

Reply:
Another option is leave them fail, at the end there are just 2 installers broken: git and kubernetes-tools, until the PRs will be merged. If it’s ok with you, we can proceed like this.

Reply:
I thought that would break the build - but sure - if that works - I'm fine with that!